app/assets/stylesheets/components/_figure.scss

```diff
@@ -36,3 +36,11 @@
     margin-top: $gutter-one-third;
   }
 }
+
+.app-c-figure__figcaption-text {
+  margin-bottom: $gutter-one-third;
+
+  @include media(mobile) {
```

Comment:
We normally try and do mobile-first development, which means our CSS should be based on mobile devices and we should add any other styling for larger screen sizes in media queries. So in this case, it would mean swapping round the CSS you've added here.

Reply:
@vanitabarrett Updated and fixed up.